src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx

```diff
@@ -0,0 +1,546 @@
+---
+title: Monitor your .NET MAUI mobile application
+tags:
+    - .NET MAUI
+    - Mobile monitoring
+    - Hybrid framework
+    - Get started
+metaDescription: "New Relic's .NET MAUI agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+---
+
+import mobileFlutterSummary from 'images/mobile_screenshot-full_flutter-summary.webp'
+
+Our New Relic .NET MAUI agent monitors your .NET MAUI mobile app and provides deep insights into your app's performance, errors, and user experience. With the agent, you can:
+
+* Capture C# errors to quickly identify and fix problems.
+* Track network requests to see how your app is interacting with the backend.
+* Use distributed tracing to drill down into handled exceptions and identify the root cause of problems.
+* Create custom tracking events and metrics to fully understand how your users are interacting with your app.
+
+<img
+  title="Mobile summary view in the UI"
+  alt="Summary view of a .NET MAUI mobile app in New Relic"
+  src={mobileFlutterSummary}
+  style={{ maxWidth: '70%' }}
+/>
+
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select your .NET MAUI app) > Summary**: View .NET MAUI data, track HTTP requests and errors, and monitor how your app is performing over time.
+</figcaption>
+
+## Install the .NET MAUI SDK
+<Steps>
+
+<Step>
+### Review the requirements
```

Comment:
we still want to do the [#requirements] anchor link for these even when in steps, right?